src/VisualStudio/Next/Remote/ServiceHubRemoteHostClient.cs

```diff
@@ -16,9 +16,10 @@ namespace Microsoft.VisualStudio.LanguageServices.Remote
 {
     internal partial class ServiceHubRemoteHostClient : RemoteHostClient
     {
-        private readonly HubClient _hubClient;
         private readonly Stream _stream;
-        private readonly JsonRpc _rpc;
+
+        protected readonly HubClient _hubClient;
+        protected readonly JsonRpc _rpc;
```

Comment:
Note:  These accessibility changes are so that I can create my own derived ```ServiceHubRemoteHostClient```.  I will cc everyone on that review as well.